style.css

```diff
@@ -0,0 +1,259 @@
+    margin: 20px 0;
+    align-items: flex-start;
+}
+section .container .contactinfo .info li span:nth-child(1){
+    width: 30px;
+    min-width: 30px;
+}
+section .container .contactinfo .info li span:nth-child(1) img{
+    max-width: 100%;
+    filter: invert(1);
+    opacity: 0.5;
+}
+section .container .contactinfo .info li span:nth-child(2){   
+    color: #fff;
+    margin-left: 10px;
+    font-weight: 300;
+    opacity: 0.7;
+}
+section .container .contactinfo .info li:hover span:nth-child(1) img,
+section .container .contactinfo .info li:hover span:nth-child(2){
+    opacity: 1;
+}
+section .container .contactinfo .sci{
+    position: relative;
+    display: flex;
+}
+section .container .contactinfo .sci li{
+    list-style: none;
+    margin-right: 15px;
+}
+section .container .contactinfo ul.sci li a{
+    text-decoration: none;
+}
+section .container .contactinfo .sci li a img{
+    filter: invert(1);
+    opacity: 0.5;
+}
+section .container .contactinfo .sci li:hover a img{
+    opacity: 1;
+}
+section .container .contactform{
+    position: absolute;
+    padding: 70px 50px;
+    background: #fff;
+    margin: 0 0 0 150px;
+    padding: 0 0 0 250px;
+    width: calc(100% - 150px);
+    height: 100%;
+    box-shadow: 0 50px 50px rgba(0, 0, 0, 0.5);
+}
+section .container .contactform h2{
+    color: #0f3950;
+    font-size: 24px;
+    font-weight: 500;
+}
+section .container .contactform .formbox{
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    padding: 30px 0 0 0;
+}
+section .container .contactform .formbox .inputbox{
+    position: relative;
+    margin: 0 0 35px 0;
+}
+section .container .contactform .formbox .inputbox.w50{
+    width: 47%;
+}
+section .container .contactform .formbox .inputbox.w100{
+    width: 100%;
+}
+section .container .contactform .formbox input{
+    width: 100%;
+    padding: 5px 0;
+}
+section .container .contactform .formbox .inputbox input,
+section .container .contactform .formbox .inputbox textarea
+{
+    width: 100% !important;
+    padding: 5px;
+    resize: none;
+    font-size: 17px;
+    font-weight: 300;
+    color: #333;
+    border: none;
+    border-bottom: 1px solid #777;
+    outline: none;
+}
+section .container .contactform .formbox .inputbox textarea{
+    min-height: 120px;
+}
+section .container .contactform .formbox .inputbox span{
+    position: absolute;
+    left: 0;
+    padding: 5px 0;
+    font-size: 18px;
+    font-weight: 300;
+    color: #333;
+    transition: 0.5s;
+    pointer-events: none;
+}
+section .container .contactform .formbox .inputbox textarea:focus ~ span,
+section .container .contactform .formbox .inputbox input:focus ~ span,
+section .container .contactform .formbox .inputbox input:valid ~ span,
+section .container .contactform .formbox .inputbox textarea:valid ~ span
+{
+    transform: translateY(-20px);
+    font-size:12px;
+    font-weight: 400;
+    letter-spacing: 1px;
+    color: #ff569c;
```

Comment:
group similar property...
like this : 
transform: translateY(-20px);

    font-size:12px;
    font-weight: 400;

    letter-spacing: 1px;

    color: #ff569c;

Reply:
ok thanks for suggestion